src/sentry/static/sentry/app/views/releaseDetails.jsx

```diff
@@ -132,10 +132,10 @@ const ReleaseDetails = React.createClass({
               </div>
             </div>
             <ul className="nav nav-tabs">
-              <ListLink to={`/${orgId}/${projectId}/releases/${encodeURIComponent(release.version)}/`} isActive={(to)=> {
+              <ListLink to={`/${orgId}/${projectId}/releases/${encodeURIComponent(release.version)}/`} isActive={(loc) => {
                 // react-router isActive will return true for any route that is part of the active route
                 // e.g. parent routes. To avoid matching on sub-routes, insist on strict path equality.
-                return to === this.context.location.pathname;
+                return loc.pathname === this.props.location.pathname;
```

Comment:
yeah, [upgrade guide](https://github.com/ReactTraining/react-router/blob/master/upgrade-guides/v2.0.0.md#accessing-location) and [docs](https://github.com/ReactTraining/react-router/blob/v2.8.1/docs/API.md#route-components) for more details